virttest/libvirt_xml/vm_xml.py

```diff
@@ -1749,11 +1748,7 @@ def __init__(self, virsh_instance=base.virsh):
                                  forbidden=[],
                                  parent_xpath='/',
                                  tag_name='topology')
-        accessors.XMLElementList(property_name="numa_cell",
```

Comment:
if numa_cell propery is removed from vm_xml, some cases in tp-libvirt, which is dependent on it, may fail.

Reply:
The property can be added as required.Since the xml in numa/cell is becaming more complicated for new feature added,  it's need to be defined as a class.

Reply:
Related tp-libvirt test scripts are changed in PR ttps://github.com/autotest/tp-libvirt/pull/2976